components/auth/LoginMenu.tsx

```diff
@@ -13,7 +13,8 @@ const LoginMenu = ({ isAuthenticated }: LoginMenuProp) => {
   const { i18n, t } = useTranslation("common");
   const handleClick = () => {
     clearTemplateStore();
-    signOut({ callbackUrl: `/${i18n.language}/auth/logout` });
+    signOut({ redirect: false, callbackUrl: "/" });
```

Comment:
Can we use `${window.location.origin}/${i18n.language}/auth/logout` ?

Reply:
Sadly, that didn't work ... nextAuth removes the domain prefix and replaces it with the base url set in NEXTAUTH_URL in the environment

Reply:
Just adding this here per Slack discussion https://next-auth.js.org/configuration/callbacks#redirect-callback